components/extensions/src/main/kotlin/io/github/composegears/valkyrie/extensions/IconType.kt

```diff
@@ -1,10 +1,9 @@
-package io.github.composegears.valkyrie.parser
+package io.github.composegears.valkyrie.extensions
 
 import java.nio.file.Path
 import kotlin.io.path.extension
 
-@PublishedApi
-internal enum class IconType(val extension: String) {
+enum class IconType(val extension: String) {
```

Comment:
or can be moved only `String?.isSvg` extensions

Reply:
Looks good now. Prefer placing them in a file.